app/helper.R

```diff
@@ -114,18 +114,15 @@ make_gt_from_table <- function(current_table, row_to_highlight, table_title) {
 
 # Print out contact information for the bottom of the app
 print_contact_info <- function() {
-  daslWebsite <- a("Data Science Lab (DaSL)", href="https://hutchdatascience.org")
-  daslTA <- a("Translational Analytics", href="https://hutchdatascience.org/tr-analytics/")
-  daslEmail <- a("analytics@fredhutch.org", href="mailto:analytics@fredhutch.org")
+  issue_tracker <- a("a new issue", href="https://github.com/FredHutch/trm-calculator/issues/new/")
+  email <- a("katrus@fredhutch.org", href="mailto:katrus@fredhutch.org")
   
   HTML(
     paste0(
-      "This application was developed by the Fred Hutch ",
-      daslWebsite,
-      ". For questions or feedback regarding this application, email DaSL ",
-      daslTA,
-      " at ",
-      daslEmail,
+      "This application was developed in partnership with the Fred Hutch Data Science Lab (DaSL). To request new features or report any concerns, please file ",
```

Comment:
Thanks for changing this.

Do you know if you'll get notified by email to your Fred Hutch email address if someone creates an issue?

Reply:
Great point. I updated my "Watch" settings for the repo to get notified about new issues / PRs.